spanner/integration_test.go

```diff
@@ -3177,6 +3256,7 @@ func TestIntegration_BatchDML_Error(t *testing.T) {
 }
 
 func TestIntegration_PGNumeric(t *testing.T) {
+	skipUnsupportedPGTest(t)
```

Comment:
This test would only work for PG but not GoogleSQL. We can to add a `skipUnsupportedTestForGoogleSQL` and call it here.

Reply:
Or something like, `onlyRunForPGTest()`. If dialect is not PG, then skip.